test/client_endpoint_test.go

```diff
@@ -136,11 +147,11 @@ func (f *NsmgrTestSuite) TestNSmgrEndpointSendFD() {
 
 	f.registerCrossNSE(ctx, setup, nseRegClient, t)
 
-	cl := client.NewClient(context.Background(),
+	cl := retry.NewClient(client.NewClient(context.Background(),
```

Comment:
Why is the NSMgr doing retry... I would expect only the NSC to do retry.

Reply:
Oops, this can be removed

Reply:
Ah... this is in a test :)

Reply:
yes, it is a test, and also retry is not needed actually here. It should work without retries.

Reply:
Forgot to remove debug stuff